docs/source/examples/substitutions.py

```diff
@@ -0,0 +1,7 @@
+# adapted from t_sub.py / t_NomialSubs / test_Basic
+from gpkit import Variable
+x = Variable("x")
+p = x**2
+assert p.sub(x, 3) == 9
```

Comment:
```
Traceback (most recent call last):
  File "substitutions.py", line 5, in <module>
    assert p.sub(x, 3) == 9
  File "/home/matthew/.local/lib/python3.6/site-packages/gpkit/nomials/math.py", line 171, in sub
    return Signomial(self.hmap.sub(substitutions, self.varkeys),
  File "/home/matthew/.local/lib/python3.6/site-packages/gpkit/nomials/map.py", line 107, in sub
    fixed, _, _ = parse_subs(varkeys, substitutions)
  File "/home/matthew/.local/lib/python3.6/site-packages/gpkit/nomials/substitution.py", line 23, in parse_subs
    for var in substitutions:
TypeError: 'Variable' object is not iterable
```

Reply:
An easy fix is:
``` python
p.sub({x:3}).value == 9
```. The subs are always dicts.

Reply:
oops yup it looks like a lot of these examples are from when I had a little "convenience" syntax...that nobody but me used...so I removed it.